Tools/ardupilotwaf/boards.py

```diff
@@ -192,7 +198,7 @@ def configure_env(self, cfg, env):
             'm',
         ]
 
-        cfg.check_librt(env)
+        cfg.check_librt(env, mandatory=False)
```

Comment:
See my other comment. IMO we actually have to rename the check, checking for the availability of clock_gettime(), with or without librt. On MacOS we don't use this function (because it doesn't exist)... so we adopt your second option, checking DEST_OS